ws-nextjs-app/pages/[service]/send/[id]/Uploading/index.tsx

```diff
@@ -0,0 +1,16 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import Heading from '#app/components/Heading';
+import Paragraph from '#app/components/Paragraph';
+import styles from './styles';
+
+export default function Uploading() {
+  return (
+    <>
+      <Heading level={1} id="content" tabIndex={-1} css={styles.heading}>
+        Uploading
+      </Heading>
+      <Paragraph>Please wait until it is finished.</Paragraph>
```

Comment:
Translations - is this a future piece of work?

Reply:
Yea it will be a separate task for all areas of the Form.

Reply:
Nice, thanks!